### PR TITLE
OBS-361: Remove tini from all Docker containers

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,4 +30,5 @@ services:
       - start
       - --host-port=0.0.0.0:5010
     ports:
-      - "${EXPOSE_GCS_EMULATOR_PORT:-5010}:5010"
+      - "${EXPOSE_PUBSUB_EMULATOR_PORT:-5010}:5010"
```
Comment (relud) [inline docker-compose.yml:33]: noticed while editing that this was using the wrong env var name, this is the pubsub emulator

File: docker-compose.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ services:
     ports:
       - "${EXPOSE_SENTRY_PORT:-8090}:8090"
     command: run --host 0.0.0.0 --port 8090
+    stop_signal: SIGINT
```
Comment (willkg) [inline docker-compose.yml:10]: If we're going to do this, we should update `docker/images/fakesentry/Dockerfile`, too. I think we can copy this:

https://github.com/mozilla-services/socorro/blob/main/docker/Dockerfile.fakesentry

**Review by willkg — APPROVED** [review]
r+wc.

File: docker-compose.yml (hunk 2 above)
Comment (relud) [reply docker-compose.yml:10]: i think this PR already includes that change?

Comment (willkg) [reply docker-compose.yml:10]: Whoops--you're right! Not sure how I missed it.

**Pull request merged**